packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { console2 as console } from "forge-std/console2.sol";
+import { stdJson } from "forge-std/StdJson.sol";
+
+import { Safe } from "safe-contracts/Safe.sol";
+import { OwnerManager } from "safe-contracts/base/OwnerManager.sol";
+import { GuardManager } from "safe-contracts/base/GuardManager.sol";
+
+import { Deployer } from "scripts/Deployer.sol";
+
+import { LivenessGuard } from "src/Safe/LivenessGuard.sol";
+import { LivenessModule } from "src/Safe/LivenessModule.sol";
+
+import { Deploy } from "./Deploy.s.sol";
+
+struct SafeConfig {
+    uint256 threshold;
+    address[] owners;
+}
+
+struct SecurityCouncilConfig {
+    SafeConfig safeConfig;
+    uint256 livenessInterval;
+    uint256 thresholdPercentage;
+    uint256 minOwners;
+    address fallbackOwner;
+}
+
+/// @title Deploy
+/// @notice Script used to deploy and configure the Safe contracts which are used to manage the Superchain,
+///         as the ProxyAdminOwner and other roles in the system.
+contract DeployOwnership is Deploy {
+    /// @notice Internal function containing the deploy logic.
+    function _run() internal override {
+        console.log("start of Ownership Deployment");
+        deployAndConfigureFoundationSafe();
+        deployAndConfigueSecurityCouncilSafe();
+        console.log("deployed Security Council Safe!");
+        console.log("Ownership contracts completed");
+    }
+
+    /// @notice Returns a SafeConfig with similar to that of the Foundation Safe on Mainnet.
+    function _getExampleFoundationConfig() internal returns (SafeConfig memory safeConfig_) {
+        address[] memory exampleFoundationOwners = new address[](7);
+        for (uint256 i; i < exampleFoundationOwners.length; i++) {
+            exampleFoundationOwners[i] = makeAddr(string(abi.encode(i)));
+        }
+        safeConfig_ = SafeConfig({ threshold: 5, owners: exampleFoundationOwners });
+    }
+
+    /// @notice Returns a SafeConfig with similar to that of the Security Council Safe on Mainnet.
+    function _getExampleCouncilConfig() internal returns (SecurityCouncilConfig memory councilConfig_) {
+        address[] memory exampleCouncilOwners = new address[](13);
+        for (uint256 i; i < exampleCouncilOwners.length; i++) {
+            exampleCouncilOwners[i] = makeAddr(string(abi.encode(i)));
+        }
+        SafeConfig memory safeConfig = SafeConfig({ threshold: 10, owners: exampleCouncilOwners });
+        councilConfig_ = SecurityCouncilConfig({
+            safeConfig: safeConfig,
+            livenessInterval: 24 weeks,
+            thresholdPercentage: 75,
+            minOwners: 8,
+            fallbackOwner: mustGetAddress("FoundationSafe")
+        });
+    }
+
+    /// @notice Deploys a Safe with a configuration similar to that of the Foundation Safe on Mainnet.
+    function deployAndConfigureFoundationSafe() public returns (address addr_) {
+        address safe = deploySafe("FoundationSafe");
```

Comment:
yeah... I really wish it would just be a no-op to call startBroadcast inside a broadcast. 🤷

Reply:
Adding the comment in locally.

Reply:
> yeah... I really wish it would just be a no-op to call startBroadcast inside a broadcast

Only allowing this is the two signers are the same would be a reasonable feature request. I think the current restriction is to prevent accidental overwrites when broadcasting from two or more deployers